README.md

```diff
@@ -73,45 +73,73 @@ During a sprint we track progress using a [scrum or kanban-like board](#notes)<s
 
 After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
 
+The following sections provide more detail on specific phases of sprint planning mentioned above. 
+
+##Task Estimation 
+
+Each task requires an estimated point value based on time, complexity, and unknowns. Ensuring tickets are broken down into sensible, actionable chunks can reduce unknowns and allow the team to point a ticket based on the task’s time and complexity. A good scale to use for point values is the fibonacci sequence (1, 2, 3, 5, 8, 13) [because it reflects that uncertainty on how to execute a task increases with task size](http://www.scrum-institute.org/Effort_Estimations_Planning_Poker.php).
+
+A team should point each ticket as if any member were to take it on. Team members should therefore estimate each task as a group, and converge on a point value on which every member agrees. One method is planning poker, in which each team member reveals their individual point estimation of a task at the same time before the team collectively decides on the appropriate point value. 
+
+It does not matter if two teams use different pointing scales or other practices specific to their pointing process, as long as they are consistent within their own team. For example, a team might decide to always re-assess tickets that were assigned a point value of 13, because that suggests the task has not been understood and broken down sufficiently. 
+
 ## Velocity Tracking
 
-### What is Velocity Tracking?
+Velocity tracking uses a team's average historical output to inform how much work (measured in total number of points) to commit to in the next sprint. Velocity tracking can also help a team reflect on how much work was accomplished during retrospective by comparing their *estimated* sprint velocity to their *actual* sprint velocity, often [visualized with a burn-down chart](https://en.wikipedia.org/wiki/Burn_down_chart).
+
+### What is Sprint-Velocity?
+
+`Sprint-velocity` describes the rate at which work is completed over the duration of a sprint. In other words, sprint-velocity reflects the number of points completed by an individual team member in one hour (or points per person-hour).
```

Comment:
@kvenn I totally agree this can make the reader think this is about per person evaluation when it's not - I like that phrasing better